### ISSUE TITLE
Not able to delete or move Sections

### ISSUE THREAD

**Issue by Ryan529** [ThemeFuse/Unyson#1490]
I'm not able to move, or delete sections. Every time I do, then save, they reappear. They also have the function code displayed at the top of each section instead of the word "Section."
<img width="685" alt="screen shot 2016-04-19 at 4 32 46 pm" src="https://cloud.githubusercontent.com/assets/1326552/14658929/7a4ab398-064c-11e6-87f6-cc711f674dc1.png">

**Comment by duongancol** [comment]
we have the same issue. All WP, Unyson, Page-builder are the latest version.

**Comment by moldcraft** [comment]
Go to Updates page, press Check Now, see if there are new extension updates available, do Update.

**Comment by duongancol** [comment]
Please don't close this issue, 
As I said: https://github.com/ThemeFuse/Unyson/issues/1490#issuecomment-212349458
You could see [here](http://d.pr/i/18c9L) and [here](http://d.pr/i/1lQdn)

**Comment by moldcraft** [comment]
@duongancol can you give me admin/ftp access?

**Comment by duongancol** [comment]
I did, Thanks!

**Comment by moldcraft** [comment]
@duongancol I logged in and it works fine

![](https://static.md/42d3ba1ab3b8bf1e72db406b07179a83.png)

Try to Ctrl+R in browser.

**Comment by Ryan529** [comment]
For a while I wasn't able to update unysom as I kept getting errors there.
I disabled all my plugins and started re-adding them one by one. File
manager seemed to be causing the issue, at least for now.

This was on my localhost, as I didn't want to recreate it on my live site.

Ryan Anderson
206.547.0954
On Apr 20, 2016 6:48 AM, "moldcraft" <notifications@github.com> wrote:

@duongancol <https://github.com/duongancol> I logged in and it works fine

<https://camo.githubusercontent.com/51a79f1ff9d1fc9ce7a44207d0b8a80ea052fc3f/68747470733a2f2f7374617469632e6d642f34326433626131616233623862663165373264623430366230373137396138332e706e67>

Try to Ctrl+R in browser.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/ThemeFuse/Unyson/issues/1490#issuecomment-212432569>
